docs/toml-schema.md

```diff
@@ -58,9 +58,16 @@ alumni = [
 # Optional, name of other teams whose members will be included as members of this team.
 # Defaults to empty.
 included-teams = []
-# Optional, include all members of all other teams.
-# Defaults to false.
+# Include all members of all other teams. Optional, defaults to false.
 include-all-team-members = false
+# Include all team leads. Optional, defaults to false.
+include-team-leads = false
+# Include all working group leads. Optional, defaults to false.
+include-wg-leads = false
+# Include all project group leads. Optional, defaults to false.
+include-project-group-leads = false
+# Include all alumni. Optional, defaults to false.
+include-all-alumni = false
```

Comment:
This aren't really meant to be used except by "special" teams (e.g., `include-all-alumni` should only be used by the alumni team). Can we mark these as not for use somehow?

Reply:
Sure, I had thought of including something but wasn't sure.